presto-orc/src/main/java/com/facebook/presto/orc/metadata/statistics/BloomFilter.java

```diff
@@ -13,12 +13,10 @@
  */
 package com.facebook.presto.orc.metadata.statistics;
 
-import org.apache.hive.common.util.Murmur3;
+import com.google.common.annotations.VisibleForTesting;
 
-import java.io.DataInputStream;
 import java.io.DataOutputStream;
 import java.io.IOException;
-import java.io.InputStream;
 import java.io.OutputStream;
 import java.util.Arrays;
 
```

Comment:
We can also add a comment to mention that this class was forked from org.apache.hive.common.util.BloomFilter.